aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -215,6 +215,18 @@ module aptos_framework::fungible_asset {
         frozen: bool,
     }
 
+    #[event]
+    struct GasBurnt has drop, store {
```

Comment:
we are not replacing Burn event, we are replacing Withdraw event, so this should be `GasWithdraw` or `WithdrawForGas`, similar below.

Reply:
There was no burn event. This event is for gas burn... I think.

Reply:
there is no burn event, so this isn't an equivalent of Burn just for gas, this is equivalent of Withdraw just for gas. 

so this should be `GasWithdraw` :) same below - it is `GasDeposit`

(or WithdrarGasFee and DepositGasRefund)